Specs/Scene/Cesium3DTilesetSpec.js

```diff
@@ -2051,9 +2052,9 @@ defineSuite([
             });
 
             expect(renderOptions).toRenderAndCall(function(rgba) {
-                expect(rgba[0]).toBeGreaterThan(0);
-                expect(rgba[1]).toEqual(0);
-                expect(rgba[2]).toEqual(0);
+                expect(rgba[0]).toBeGreaterThan(200);
+                expect(rgba[1]).toBeLessThan(25);
+                expect(rgba[2]).toBeLessThan(25);
```

Comment:
The colors are updated since pbr shading doesn't produce the same sort of solid shading as before.